modules/simple_forms_api/app/controllers/simple_forms_api/v1/scanned_form_uploads_controller.rb

```diff
@@ -103,6 +103,16 @@ def perform_pdf_upload(location, file_path, metadata)
           upload_url: location
         )
       end
+
+      def check_for_changes
+        in_progress_form = InProgressForm.form_for_user('FORM-UPLOAD-FLOW', @current_user)
+        if in_progress_form
+          prefill_data_service = SimpleFormsApi::PrefillDataService.new(prefill_data: in_progress_form.form_data,
+                                                                        form_data: params[:form_data],
```

Comment:
Does params[:form_data] only contain prefill data or are there blank fields in there too that the user has to input?

Reply:
The user can input a few fields but most of them, I think, can be prefilled.